### ISSUE TITLE
Fedora32 - basically no actions can be executed

### ISSUE THREAD

**Issue by klslz** [sandrotosi/puddletag#6]
Thx for your efforts to migrate puddletag. It's been the best tagger for years.

I'm running Fedora32. It comes with python3 and Wayland support.

The installation of dependencies would look like this on Fedora 32:

`dnf install python python-six  python-mutagen python-configobj python-pyparsing python-qt5 qt5-qtsvg`

python3 is default btw. the package names do not need a python3-... anymore.

Wayland is also default on Fedora 32 btw..

puddletag, after cloning, starts up as you described. Great. 
However. Wayland doesn't support QWindow as it seems. Anytime I hover over the menues I get a 

> qt.qpa.wayland: Wayland does not support QWindow::requestActivate()

I  then logged in into X11. QWindow erros were gone.

Basically all I can do under both Window managers is to load a music directory and edit fields. 
Pretty much non of the submenues or shortcuts works, I can't save the changes, nothing.

Anything I might be missing? 

Thx.

**Comment by sandrotosi** [comment]
@klslz Marko recent work on puddletag should have addressed the issues you were having on fedora, could you please give it a new try to the latest HEAD? thanks!

Marko also suggested that the wayland warning can be safely ignored: https://github.com/lupoDharkael/flameshot/issues/619#issuecomment-549668035

**Comment by klslz** [comment]
Just tested the latest updates

```TARGET=/tmp/puddletag
test -d $TARGET && rm -rf $TARGET
git clone --depth=1 --branch=pyqt5+py3 https://github.com/sandrotosi/puddletag $TARGET
PYTHONPATH=$TARGET/source/ $TARGET/source/puddletag
```

Everything related to this issue report is OK.

Files can be loaded, edited and saved. Filenames can also be changed.
Tested under Wayland and Fedora32.

I'll do some more testing and report.

Thx for now - the progress looks promising.
